lib/pages/chat/chat_event_list.dart

```diff
@@ -145,6 +146,9 @@ class ChatEventList extends StatelessWidget {
                   final nextEvent = index < controller.timeline!.events.length
                       ? controller.timeline!.events[currentEventIndex + 1]
                       : null;
+
+                  _cleanCancelTokenMap(event);
+
```

Comment:
I still not get your point? why put it here the cleanup?

Reply:
clean up moved to `chat.dart`